recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domu-nfs-no-coproc.cfg

```diff
@@ -0,0 +1,34 @@
+# =====================================================================
+# Example PV Linux guest configuration
+# =====================================================================
+#
+# This is a fairly minimal example of what is required for a
+# Paravirtualised Linux guest. For a more complete guide see xl.cfg(5)
+
+seclabel='system_u:system_r:domU_t'
+
+# Guest name
+name = "DomU"
+
+# Kernel image to boot
+kernel = "/boot/domu/Image"
+
+device_tree = "/xen/domu.dtb"
```

Comment:
AFAIK he uses DT for passing "pvrkm" node to the guest taking advantage of the hole in libxl (all stuff under passthrough directory are being copied as is).